src/Button.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function Button({ submitType, onClick, children }) {
+  return (
+    <button type={submitType ? 'submit' : 'button'} onClick={onClick}>{children}</button>
```

Comment:
너무 길어지면 줄바꿈을 하는게 더 가독성이 좋습니다 :)
```suggestion
    <button 
      type={submitType ? 'submit' : 'button'} 
      onClick={onClick}
    >        
      {children}
    </button>
```